xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include "platform/win32/CharsetConverter.h"
 #include "platform/win32/WIN32Util.h"
+#include "platform/win32/WinRtUtil.h"
```

Comment:
After remove `CWinRtUtil::GetWindowsHDRStatus()` call this header is not needed

Reply:
correct